test-tools/dynamic-aot-debug/README.md

```diff
@@ -0,0 +1,59 @@
+# Dynamic AOT Module Debugging
+
+### How to use
+
+#### 1. Enable config
+
+```
+WASM_ENABLE_AOT=1
+WASM_ENABLE_DYNAMIC_AOT_DEBUG=1
+```
+
+#### 2. Build aot and obj file
+
+- build [wamrc](https://github.com/bytecodealliance/wasm-micro-runtime/tree/main/wamr-compiler) (enable WAMR_BUILD_DEBUG_AOT)
+- build test.c to test.wasm
+
+```
+/opt/wasi-sdk/bin/clang -O0 -nostdlib -z stack-size=8192 -Wl,--initial-memory=65536 -g -gdwarf-2   -o test.wasm test.c -Wl,--export=main -Wl,--export=__main_argc_argv  -Wl,--export=__heap_base -Wl,--export=__data_end -Wl,--no-entry -Wl,--allow-undefined
+```
+
+- build test.wasm to test.aot
+
+Compile and generate the test.aot file using wamrc without WAMR_BUILD_DEBUG_AOT.
```

Comment:
not sure why is this step needed?

Reply:
(⊙o⊙)…
because the WAMR_BUILD_DEBUG_AOT macro wasn't enabled when compiling iwasm. If you enable it now, it might cause issues when running test.aot with iwasm.